src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -472,6 +512,8 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
               case _ => None
             }
 
+          case Some(_: ap.PredExprInstance) => ??? // TODO
```

Comment:
This would be another task for the second PR in which you're adapting the constructor's syntax